docs/3.0.0-beta.x/concepts/configurations.md

```diff
@@ -450,6 +450,7 @@ The session doesn't work with `mongo` as a client. The package that we should us
 - `admin`
   - `autoOpen` (boolean): Enable or disabled administration opening on start (default: `true`)
   - `path` (string): Allow to change the URL to access the admin (default: `/admin`).
+  - `watchIgnoreFiles` (array): Add custom files that should not be watched during development (default: `[]`)
```

Comment:
can you just link to the chokidar do for the pattern documentation ;)

Reply:
Do you think of anything particular or is it enough to just provide this link https://github.com/paulmillr/chokidar? I couldn't find docs :/ @alexandrebodin

Reply:
You should link to this https://github.com/paulmillr/chokidar#path-filtering
specifying it uses the ignored param of chokidar :)

Reply:
take a look now, please :)